### DISCUSSION TITLE
Pivpn encryption

### DISCUSSION THREAD

**Discussion by J4ME51** [pivpn/pivpn#1469] [Q&A]
Why does wireguard only use 128bit encryption and not 256Bit?
Unless I have misunderstood surely OpenVPN would be better 

If you go for WireGuard, you don't get to choose: you will use a Curve25519 public key, which provides 128-bit security

**Comment by TinCanTech** [comment]
Elliptic curve cryptography is specifically designed to require less bits.

Edwards curves (ED25519, ED448) are, to my understanding, completely _predefined_ ..

You have to learn a lot about cryptography before you _understand_.
